src/sync/SyncQueue.js

```diff
@@ -71,7 +71,9 @@ export class SyncQueue {
                                 recordType,
                                 record.id)
                               .length > 0;
-          if (!duplicate) {
+          if (duplicate) {
+            duplicate.changeTime = new Date().getTime();
```

Comment:
Just looking at this again, it's a good example of why variables should provide an indication of booleanness by being called things like isDuplicate